apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -43,6 +44,7 @@ li.ui-state-highlight {
   list-style-type: none;
   background-color: #90a4ae !important;
   min-width: 3em !important;
+  margin-bottom: 5px !important;
```

Comment:
do we need the `!important` here? its kind of hack and should be avoided whenever possible.

Reply:
Ah, yeah that needs to go and is now removed. Sorry, Copilot has a tendency to autocomplete those in and I sometime forget to remove them. 😞